tests/0.8.1/valid.json

```diff
@@ -0,0 +1,376 @@
+        ]
+      }
+    }
+  },
+  {
+    "comment": "Delegated UCAN can delegate",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwV1BFZncyeVVYdG1NejZTVzRoeWU3WHBqQlFTTjFFV2plNnQxMm1SREM3ViIsImF1ZCI6ImRpZDprZXk6ejZNa3BrUVBYSHV4b3FIN291a2RvaFhiUTh1VThRRThLcEFpdVlwQ0Vod0J2aGZmIiwiZXhwIjo0ODAzODkwNTQ0LCJhdHQiOlt7IndpdGgiOiJwcmYvMCIsImNhbiI6InVjYW4vREVMRUdBVEUifV0sInByZiI6WyJleUpoYkdjaU9pSkZaRVJUUVNJc0luUjVjQ0k2SWtwWFZDSXNJblZqZGlJNklqQXVPQzR4SW4wLmV5SnBjM01pT2lKa2FXUTZhMlY1T25vMlRXdGxhMjV5Vmsxa1VXSTJRa2cxZGtJMFZUWjBZbWc0UnpWTmRsTkZkekpsY1hCVVkybG9PRmhGWWsxM1J5SXNJbUYxWkNJNkltUnBaRHByWlhrNmVqWk5hM0JYVUVWbWR6SjVWVmgwYlUxNk5sTlhOR2g1WlRkWWNHcENVVk5PTVVWWGFtVTJkREV5YlZKRVF6ZFdJaXdpWlhod0lqbzBPREF6T0Rrd05UUTBMQ0poZEhRaU9sdGRMQ0p3Y21ZaU9sdGRmUS5oMmxobWtLUDRWdzFHZzFYVGFTZ0JaMURNamZDUzBiNlcybmlodFJMelBXS3pFSzBHQjc4a1pKLUE5NUk3aFh3c0lKUDZBNlgzSUhVOEZFVW9rem1DdyJdfQ.BYLOS5dAB9bSMBYgPEQXKZIO_7mMi0DpJrOZUw689QPTwJhc2Foqe7DbZ2bwPPuNOCK-9YfRfATquqrI8iI2CQ",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "exp": 4803890544,
+        "att": [
+          {
+            "with": "prf/0",
+            "can": "ucan/DELEGATE"
+          }
+        ],
+        "prf": [
+          "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtla25yVk1kUWI2Qkg1dkI0VTZ0Ymg4RzVNdlNFdzJlcXBUY2loOFhFYk13RyIsImF1ZCI6ImRpZDprZXk6ejZNa3BXUEVmdzJ5VVh0bU16NlNXNGh5ZTdYcGpCUVNOMUVXamU2dDEybVJEQzdWIiwiZXhwIjo0ODAzODkwNTQ0LCJhdHQiOltdLCJwcmYiOltdfQ.h2lhmkKP4Vw1Gg1XTaSgBZ1DMjfCS0b6W2nihtRLzPWKzEK0GB78kZJ-A95I7hXwsIJP6A6X3IHU8FEUokzmCw"
+        ]
+      }
+    }
+  },
+  {
+    "comment": "UCAN is valid",
+    "token": "eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCIsInVjdiI6IjAuOC4xIn0.eyJpc3MiOiJkaWQ6a2V5Ono2TWtwV1BFZncyeVVYdG1NejZTVzRoeWU3WHBqQlFTTjFFV2plNnQxMm1SREM3ViIsImF1ZCI6ImRpZDprZXk6ejZNa3BrUVBYSHV4b3FIN291a2RvaFhiUTh1VThRRThLcEFpdVlwQ0Vod0J2aGZmIiwiZXhwIjo0ODAzODkwNTQ0LCJhdHQiOltdLCJwcmYiOltdfQ.gTVI9hmLlv0JgiRYsuZ1xPxSn6T0fvyw_u8CbjaI1TW15RsqLQQv1j2aBeHk9TY7-hqNcuPCZFdgX2GgsgB6Aw",
+    "assertions": {
+      "header": {
+        "alg": "EdDSA",
+        "typ": "JWT",
+        "ucv": "0.8.1"
+      },
+      "payload": {
+        "iss": "did:key:z6MkpWPEfw2yUXtmMz6SW4hye7XpjBQSN1EWje6t12mRDC7V",
+        "aud": "did:key:z6MkpkQPXHuxoqH7oukdohXbQ8uU8QE8KpAiuYpCEhwBvhff",
+        "exp": 4803890544,
+        "att": [],
+        "prf": []
+      }
+    }
+  },
+  {
+    "comment": "Payload `nbf` is valid",
```

Comment:
Is this test redundant with "UCAN is ready to be used"? If we are testing the same thing, we should only keep one.